src/components/Patient/Utils.ts

```diff
@@ -17,3 +25,38 @@ export function isPatientMandatoryDataFilled(patient: PatientModel) {
     patient.blood_group
   );
 }
+
+export const useAddPatientNote = (options: {
+  patientId: string;
+  consultationId?: string;
+}) => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: async ({
+      noteField,
+      thread,
+      replyTo,
+    }: {
+      noteField: string;
+      thread: PatientNotesModel["thread"];
+      replyTo?: { id: string };
+    }) => {
+      const { res, data } = await request(routes.addPatientNote, {
+        pathParams: { patientId: options.patientId },
+        body: {
+          note: noteField,
+          thread,
+          consultation: options.consultationId,
+          reply_to: replyTo?.id,
+        },
+      });
+      if (res?.status === 201 && data) return data;
```

Comment:
use `mutate` function instead of `request`. `request` is deprecated. See docs.ohc.network